ds3-autogen-net/src/main/java/com/spectralogic/ds3autogen/net/model/response/BaseResponse.java

```diff
@@ -21,15 +21,13 @@
 
 public class BaseResponse {
 
-    private final NetHelper netHelper;
+    private final NetHelper netHelper = NetHelper.getInstance();
```

Comment:
The main reason is for testability.  By injecting the implementation we can mock the implementation in a test and and then pass it in.  Unless this is something that we are not doing (though we probably should be...).